build.sh

```diff
@@ -319,6 +319,7 @@ if hasArg tests || (( ${NUMARGS} == 0 )); then
           $CMAKE_TARGET == *"SPARSE_NEIGHBORS_TEST"* || \
           $CMAKE_TARGET == *"MATRIX_TEST"* || \
           $CMAKE_TARGET == *"NEIGHBORS_TEST" || \
+          $CMAKE_TARGET == *"NEIGHBORS_ANN_CAGRA_TEST" || \
```

Comment:
Some of these lines look like this suggestion. Please verify the intended spelling. Looks like NEIGHBORS_TEST is an exception as well.
```suggestion
          $CMAKE_TARGET == *"NEIGHBORS_ANN_CAGRA_TEST"* || \
```

Reply:
Also should this list be alphabetized? It is pretty close, just a few exceptions to alphabetical order.